### PR TITLE
feat: added trending page with filter and other pages empty

### PR DESCRIPTION
**Pull request by Hamza-777**

### CODE REVIEW

File: src/Components/Utilities/NavControls.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        />
+      </Link>
+      <Link to='/playlists'>
+        <MdPlaylistAdd
+          className={`icon light-color ${
+            activeControl === 'playlist' ? 'active-control' : ''
+          }`}
+          onClick={(e) => setActiveControl('playlist')}
+        />
+      </Link>
```
Comment (Developers-codz) [inline src/Components/Utilities/NavControls.jsx:52]: suggestion - jsx is mostly repeating here so can create an array of object of every nav item and loop over them using map

**Review by Developers-codz — APPROVED** [review]
Good work on code, UI also looks great.
Can provide nav some background color instead of doing  blur the content behind

File: src/Components/Utilities/NavControls.jsx (hunk 1 above)
Comment (Hamza-777) [reply src/Components/Utilities/NavControls.jsx:52]: sure

File: src/App.js [hunk 2]
```diff
@@ -1,14 +1,22 @@
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import Home from './Components/Pages/Home';
+import Trending from './Components/Pages/Trending';
 import Footer from './Components/Utilities/Footer';
 import Navbar from './Components/Utilities/Navbar';
+import Watchlater from './Components/Pages/Watchlater';
+import History from './Components/Pages/History';
+import Playlists from './Components/Pages/Playlists';
```
Comment (mikesch803) [inline src/App.js:8]: suggestion : should import from one file only

File: src/Components/Pages/Trending.jsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    activeFilter.current = e.target.name;
+    e.target.name === 'all'
+      ? dispatchVideo({
+          type: 'GET_ALL',
+          payload: {
+            videos,
+          },
+        })
+      : dispatchVideo({
+          type: 'GET_SINGLE',
```
Comment (mikesch803) [inline src/Components/Pages/Trending.jsx:33]: here get single is not good naming, here you can name this as get cateogory

File: src/Components/Providers/VideoProvider.jsx [hunk 4]
```diff
@@ -1,10 +1,38 @@
...
+      };
+    default:
+      return state;
+  }
+};
+
 const VideoProvider = ({ children }) => {
   const [videos, setVideos] = useState([]);
+  const [videoState, dispatchVideo] = useReducer(reducer, {
+    vids: [],
```
Comment (mikesch803) [inline src/Components/Providers/VideoProvider.jsx:34]: you can give the good naming

**Review by mikesch803 — APPROVED** [review]
i put some suggestions, else  i liked the theme, and the transparent navbar, also the site is mobile responsive, good work!

**Pull request merged**